layouts/_default/single.html

```diff
@@ -7,6 +7,11 @@ <h1 class="header-title">{{ .Title }}</h1>
                 {{ $ISO_time := dateFormat "2006-01-02T15:04:05-07:00" .Date }}
                 <div class="post-meta">
                     <time datetime="{{ $ISO_time }}" itemprop="datePublished"> {{ .Date | time.Format ":date_medium" }} </time>
+                    {{ if .Params.tele_username }}
+                    <p>Author: <a href="https://telegram.me/{{ .Params.tele_username }}">{{ .Params.author }}</a></p>
+                    {{ else }}
+                    <p>Author: {{ .Params.author }}</p>
+                     {{ end }}
```

Comment:
I think a better idea is to use something like author_link instead of tele_username. Then this feature can be useful for any kind of links (not only Telegram).